packages/genesys-spark-components/src/components/stable/gux-popover/example.html

```diff
@@ -1,17 +1,26 @@
 <h1>gux-popover</h1>
+
 <div class="scroll-container">
   <div class="scroll-interior">
-    <div id="popover-target">Example Element</div>
+    <gux-button
+      id="popover-target"
+      accent="primary"
+      aria-describedby="popover-content-1"
```

Comment:
This does not make sense to me. Why would an element be described by its popover content?

Reply:
I read this as the popover content is being described by the button element. So if the user focuses on the button then the screenreader could read out additionally linked information about the popover.

Here's an example online I found for using `aria-describedby` with a button and tooltip text container that has some similarities with how I implemented `aria-describedby`:

```
   <button type="button" aria-describedby="my-tooltip">Snipping Tool</button>
    <div hidden id="my-tooltip" role="tooltip">
        It can take still screenshots of an open window,
        rectangular areas, a free-form area,
        or the entire screen.
    </div>
```

What do you think?